content/en/docs/components/pipelines/overview/caching-v2.md

```diff
@@ -7,11 +7,11 @@ weight = 41
 {{% beta-status
 feedbacklink="https://github.com/kubeflow/pipelines/issues" %}}
 
-Starting from [Kubeflow Pipelines SDK v2](https://www.kubeflow.org/docs/components/pipelines/sdk-v2/) and Kubeflow Pipelines 1.7.0, Kubeflow Pipelines supports step caching capabilities in both [standalone deployment](https://www.kubeflow.org/docs/components/pipelines/installation/standalone-deployment/) and [AI platform Pipelines](https://cloud.google.com/ai-platform/pipelines/docs).
+Starting from [Kubeflow Pipelines SDK v2](https://www.kubeflow.org/docs/components/pipelines/sdk-v2/) and Kubeflow Pipelines 1.7.0, Kubeflow Pipelines supports step caching capabilities in both [standalone deployment](https://www.kubeflow.org/docs/components/pipelines/installation/standalone-deployment/) and [AI Platform Pipelines](https://cloud.google.com/ai-platform/pipelines/docs).
```

Comment:
This is renamed to Vertex Pipelines https://cloud.google.com/vertex-ai/docs/pipelines/introduction

Could you update all? Thanks!

Reply:
Definitely. I think it makes sense to separate out this change from the others, however, since it will be a fairly large/distinct change.

There are currently at least 45 references to AI Platform Pipelines within the website and many have links to different aspects of the Google Cloud docs, console, etc. that will also need to be translated to the Vertex equivalent.

Thoughts?

Also, should it be "Vertex AI Pipelines"?

Reply:
Yes, Vertex Pipelines for short